core/utils.py

```diff
@@ -0,0 +1,348 @@
+import os
+import cv2
+import gym
+import torch
+import random
+import shutil
+import logging
+
+import numpy as np
+
+from scipy.stats import entropy
+
+
+class LinearSchedule(object):
```

Comment:
Up to you if we're keeping this but if we do, please use it.

Reply:
deleted